scripts/test_ibc_transfer.sh

```diff
@@ -10,31 +10,43 @@ NEUTRON_DIR="${NEUTRON_DIR:-../../neutron}"
 HOME="$NEUTRON_DIR/data/test-1/"
 KEY="demowallet1"
 ADMIN="neutron1m9l358xunhhwds0568za49mzhvuxx9ux8xafx2"
-BIN="neutrond"
```

Comment:
What's the reason for removing `BIN` and `GAIA_BIN` constants?

Reply:
what's the original reason for them to be defined as a constant?

Reply:
Because i don't need to change binary name everywhere if it's changed for some reason

Reply:
And how often do we see them changing? Also, you will have to check and replace these constants in every single script anyway, hence I didn't see much value in them